src/pedersen.rs

```diff
@@ -1954,7 +1958,7 @@ mod tests {
 		for i in 1..100 {
 			print!("\r\r\r{}", i);
 			commits.push(secp.commit(value + i as u64, blinding.clone()).unwrap());
-			proofs.push(secp.bullet_proof(value + i as u64, blinding.clone(), blinding.clone(), None, None));
+			proofs.push(secp.bullet_proof(value + i as u64, blinding.clone(), blinding.clone(), blinding.clone(), None, None));
```

Comment:
In this whole `test_bullet_proof_verify_multi` function, especial this line, I prefer to give `rewind_nonce` and `private_nonce` different value to validate whether it works for batching.

Reply:
Okay